projects/components/src/draggable-list/draggable-item/draggable-item.component.ts

```diff
@@ -0,0 +1,18 @@
+import { ChangeDetectionStrategy, Component, Input, TemplateRef, ViewChild } from '@angular/core';
+
+@Component({
+  selector: 'ht-draggable-item',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <ng-template #draggableItem>
+      <ng-content></ng-content>
+    </ng-template>
+  `
+})
+export class DraggableItemComponent {
```

Comment:
This could reuse common code we have for this pattern. From tab, for example
```ts
@Component({
  selector: 'ht-tab',
  changeDetection: ChangeDetectionStrategy.OnPush,
  styleUrls: ['./tab.component.scss'],
  template: CONTENT_HOLDER_TEMPLATE
})
export class TabComponent extends ContentHolder {
```

Reply:
Yes, it seems a good change

Reply:
done